checker/manual/subtyping-checker.tex

```diff
@@ -234,8 +234,110 @@ \section{Subtyping Checker example\label{subtyping-example}\label{encrypted-exam
 Also, see the example project in the \<checker/examples/subtyping-extension> directory.
 
 
+\section{Type aliases and typedefs\label{subtyping-type-alias}}
+
+A type alias or typedef is a type that shares the same representation as
+another type but is conceptually distinct from it.  For example, some
+strings in your program may be street addresses; others may be passwords;
+and so on.  You wish to indicate, for each string, which one it is, and to
+avoid mixing up the different types of strings.  Likewise, you could
+distinguish integers that are offsets from those that are absolute values.
+
+Creating a new type makes your code easier to understand by conveying the
+intended use of each variable.  It also prevents errors that come from
+using the wrong type or from mixing incompatible types in an operation.
+
+If you want to create a type alias or typedef, you have two options.
+Which one you choose depends on which operations you wish your new type to support.
+
+\begin{itemize}
+\item
+  Use the Fake Enum Checker (\chapterpageref{fenum-checker}) if you wish to
+  prevent all operations on the elements of the new type.  Suppose you
+  create a new fenum type \<CompassDirection> with \<int> values \<NORTH>,
+  \<SOUTH>, \<EAST>, and \<WEST>.  The operation \<NORTH + SOUTH> should be
```

Comment:
Thanks for pointing that out.  In that case, what is the use case for the Fenum Checker?  That it permits easier creation of type checkers (less code for the checker and for the annotations), albeit with less flexibility since it enforces one choice about the default?  Are there other benefits?  What guidance should we give to users?